arangograph/certificates.md

```diff
@@ -0,0 +1,150 @@
+It is recommended to use well known certificates, as this eases the access of a
+deployment in your browser.
+
+### Self-signed X.509 certificates
+
+**Self-signed X.509 certificates** are used on the high ports, i.e. `18529`.
+This type of certificate has a lifetime of one year and it is created by the
+ArangoGraph platform. It is also rotated automatically before the expiration
+date.
+
+{% hint 'info' %}
+Unless you switch off the **Use well known certificate** option in the
+certificate generation, both the default and high port serve the same
+self-signed certificate.
+{% endhint %}
+
+### Subject Alternative Name (SAN)
+
+The Subject Alternative Name (SAN) is an extension to the X.509 specification 
+that allows you to specify additional host names for a single SSL certificate.
+
+When using [private endpoints](deployments.html#how-to-create-a-private-endpoint-deployment),
+[notebooks](notebooks.html), and [metrics](monitoring-metrics.html), you can specify 
+alternate domain names. Note that these are added **only** to the self-signed certificate
+as Subject Alternative Name (SAN).
+
+## How to create a new certificate
+
+1. Click a project name in the **Projects** section of the main navigation.
+2. Click the **Security** tab.
+3. In the **Certificates** section, click:
+   - The **New certificate** button to create a new certificate.
+   - A name or the **eye** icon in the **Actions** column to view a certificate.
+     The dialog that opens provides commands for installing and uninstalling
+     the certificate through a console.
+   - The **pencil** icon to edit a certificate.
+     You can also view a certificate and click the **Edit** button.
+   - The **tag** icon to make the certificate the new default.
+   - The **recycle bin** icon to delete a certificate.
+
+![ArangoGraph Create New Certificate](images/arangograph-new-certificate.png)
+
+Certificates that have the **Use well known certificate** option enabled do
+not need any installation and will be supported by almost all web browsers
+automatically.
+
+When creating a self-signed certificate that has the **Use well known certificate**
+option disabled, the certificate needs to be installed on your local machine as
+well. This operation slightly varies between operating systems.
+
+![ArangoGraph Certificates](images/arangograph-cert-page-with-cert-present.png)
+
+![ArangoGraph Certificate Install Instructions](images/arangograph-cert-install-instructions.png)
+
+## How to connect to your application
+
+[ArangoDB drivers](../drivers/index.html), also called connectors, allow you to
+easily connect and manipulate ArangoGraph deployments right from within your
+application. 
+
+1. Navigate to the **Deployments** tab and click the **View** button to show the
+   deployment page.
+2. In the Quick start section, click the **Connecting drivers** button.
+3. Select your programming language, i.e. Go, Java, Python, etc.
+4. Use the code examples to connect a driver to your deployment.
```

Comment:
I think you should point out here that the example includes code for dealing with certificates. Otherwise you can only tell from the screenshot why this section about drivers is on the certificates page.

Reply:
Good point 👍